src/gui/MainPane/Room.qml

```diff
@@ -44,23 +44,23 @@ HTileDelegate {
                     color: theme.mainPane.listView.room.name
                 }
 
-                // HLabel {
-                //     text: model.mentions
-                //     font.pixelSize: theme.fontSize.small
-                //     verticalAlignment: Qt.AlignVCenter
-                //     leftPadding: theme.spacing / 4
-                //     rightPadding: leftPadding
-
-                //     scale: model.mentions === 0 ? 0 : 1
-                //     visible: scale > 0
-
-                //     background: Rectangle {
-                //         color: theme.colors.alertBackground
-                //         radius: theme.radius / 4
-                //     }
-
-                //     Behavior on scale { HNumberAnimation {} }
-                // },
+                HLabel {
+                    text: model.unreads
+                    font.pixelSize: theme.fontSize.small
+                    verticalAlignment: Qt.AlignVCenter
+                    leftPadding: theme.spacing / 4
+                    rightPadding: leftPadding
+
+                    scale: model.unreads === 0 ? 0 : 1
+                    visible: scale > 0
+
+                    background: Rectangle {
+                        color: model.mentions === 0 ? theme.colors.unreadBackground : theme.colors.alertBackground
```

Comment:
We should actually have theme properties specially for setting the indicator's color, `alertBackground` affects many other things on its own and `unreadBackground` is specific to this component.

We can have `theme.mainPane.listView.room.unreadIndicator.mentionsBackground` (defaulting to `theme.colors.alertBackground`) and `theme.mainPane.listView.room.unreadIndicator.background` (with `hsluv(188, saturation * 1.5, intensity * 52, 1)` directly) instead.